examples/kernel-dynamic-tile.cpp

```diff
@@ -15,15 +15,10 @@ int main(int RAJA_UNUSED_ARG(argc), char** RAJA_UNUSED_ARG(argv[]))
   using namespace RAJA;
 
   kernel_param<KernelPolicy<statement::Tile<
-      1,
-      tile_dynamic<1>,
-      seq_exec,
+      1, tile_dynamic<1>, seq_exec,
       statement::Tile<
-          0,
-          tile_dynamic<0>,
-          seq_exec,
-          statement::For<1,
-                         seq_exec,
+          0, tile_dynamic<0>, seq_exec,
+          statement::For<1, seq_exec,
                          statement::For<0, seq_exec, statement::Lambda<0>>>>>>>(
```

Comment:
Is there a way we can preserve some notion of nesting template statements through indentation and aligning closing '>' rather than having them all jammed on one line?

Reply:
I have been looking and haven't found anything yet.  The template argument formatting is definitely the leas configurable part of clang-format.

Reply:
Indeed. We have had issues with this in several projects. Maybe we should think about adding a script that inserts the special comments to skip clang formatting in code sections with deeply nested templates (where "deeply" is something we need to figure out how to define). We could set it up to run in a GH Actions that runs the formatting and also use it for local development.

Reply:
I am now excluding examples from the PR